interface/src/PerformanceManager.cpp

```diff
@@ -116,10 +127,12 @@ void PerformanceManager::applyPerformancePreset(PerformanceManager::PerformanceP
 
             DependencyManager::get<LODManager>()->setWorldDetailQuality(WORLD_DETAIL_LOW);
 
-        break;
-        case PerformancePreset::UNKNOWN:
+            break;
+        case
+            PerformancePreset::UNKNOWN:
+	    // Intentionally unbroken.
         default:
-            // Do nothing anymore
+            // Do nothing.
```

Comment:
Is there any reasonable way of getting here? There should either be no default for a compile-time warning, or a warning log instead.

Reply:
I don't know what UNKNOWN was intended for. I didn't really look into it but I don't remember seeing it used anywhere. It's not even the default mode. It wasn't related to this PR though so I didn't want to make too many unnecessary changes. Might be good to try logging it for a while and see if the log ever appears? If not, then remove it?

Reply:
Yeah, I think a log would be good here. I don't think anything should be reaching this point legitimately.